.travis.yml

```diff
@@ -43,15 +43,32 @@ matrix:
       env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
       script: "scripts/run_test.sh"
 
-    - scala: 2.11.4
+    - scala: 2.11.5
       env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
       script: "scripts/run_test.sh"
 
+# not committed yet
```

Comment:
Lets just move this to one of the other PR's? and remove from here -- if its not in those PR's we might forget to uncomment, and if its in them doesn't need to be here